import_export/import-wordpress.py

```diff
@@ -11,13 +11,14 @@
 from flask_restful import Api, Resource, reqparse
 from git import Repo
 
-ROOT_PATH = "/var/tms-data"
+
+ROOT_PATH = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '..'))
 
 # TODO: Unit testing, end-to-end testing
 
 def write_post(post):
     title = "wp" + str(post['id'])
-    filename = f"{ROOT_PATH}/source_files/en/{title}.json"
```

Comment:
You undid some recent fixes to ingest-wordpress when you moved it. Not a big deal, but it's good to look over the final version of a merged diff to make sure there aren't any stray changes.